test/reactionsystem_structure/reactionsystem.jl

```diff
@@ -451,18 +475,19 @@ let
         (@reaction k1, $C + D --> E + $C),
         (@reaction k2, E + $C --> $C + D)]
     @named rs = ReactionSystem(rxs, t)   # add constraint csys when supported!
-    ssys = convert(SDESystem, rs)
+    rs = complete(rs)
+    ssys = complete(convert(SDESystem, rs))
     @test issetequal(MT.get_unknowns(ssys), [B, C, D, E])
     @test issetequal(MT.get_ps(ssys), [A, k1, k2])
     du1 = zeros(4)
     du2 = zeros(4)
     sprob = SDEProblem(ssys, u0map, tspan, pmap; check_length = false)
-    sprob.f(du1, u0, p, 1.0)
+    sprob.f(du1, sprob.u0, sprob.p, 1.0)
     fs!(du2, u0, p, 1.0)
     @test isapprox(du1, du2)
     dg1 = zeros(4, 4)
     dg2 = zeros(4, 4)
-    sprob.g(dg1, u0, p, 1.0)
+    sprob.g(dg1, sprob.u0, sprob.p, 1.0)
```

Comment:
Could maybe have used test functions like `g_eval` in these cases. However, I did not want to make more changes than absolutely necessary here, as I was not fully sure what was part of the tests and what is just necessary to get there.